reference/rangefinder.md

```diff
@@ -125,6 +125,8 @@ int wb_range_finder_get_sampling_period(WbDeviceTag tag)
 
 `wb_range_finder_enable()` allows the user to enable a range-finder update each
 `ms` milliseconds.
+The provided `ms` argument specifies the sensor's sampling period.
```

Comment:
I know you didn't introduce this, but I'm wondering if it won't be better to rename the `ms` argument to `refresh_rate` or so, for all the sensors, and to indicate that its unit is in milliseconds in the comments. => The argument name should describe its purpose, not its unit.